polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -312,7 +312,9 @@ class ScalaSource[Interpreter <: ScalaInterpreter](val interpreter: Interpreter)
             .toList.sortBy(_.isPackageClass)(Ordering.Boolean.reverse)
           global.NoType -> syms
         }
-      case other => Either.left(new RuntimeException("Cannot complete tree at given position"))
+      case other =>
+        // no completions available for that tree
+        Right(global.NoType -> Nil)
```

Comment:
+1, I think this can overwrite the changes in https://github.com/polynote/polynote/pull/32/files